package-build.el

```diff
@@ -153,6 +159,26 @@ function for access to this function")
   "Remove trailing whitespace from `STR'."
   (replace-regexp-in-string "[ \t\n]*$" "" str))
 
+
+
+(defun package-build--config (package)
+  "Return the config hash for PACKAGE."
+  (or (cdr (assoc package (package-build-recipe-alist)))
+    (error "Cannot find package %s" (symbol-name package))))
+
+(defun package-build--version-to-list (str &optional regexp)
+  "Apply to STR the REGEXP if defined, \
+then pass the string to `version-to-list' and return the result."
+  (when regexp
+    (string-match regexp str)
+    (setq str (match-string 1 str)))
+  (version-to-list str))
+
+(defun package-build--valid-version-string (str regexp)
+  "Parse STR with REGEXP.
+Return the result of `package-build--version-to-list', or nil if invalid"
+  (ignore-errors (package-build--version-to-list str regexp)))
```

Comment:
Since I don't hink we need `package-build--version-to-list` as an independent function, you can inline it here.